mali: support building against 4.12

In 4.12, copy_to_user is defined in linux/uaccess.h rather than
asm/uaccess.h. Change the include from asm/uaccess.h to
linux/uaccess.h unconditionally to fix this. The include will still
work for older kernels as linux/uaccess.h includes asm/uaccess.h.

See commit 4de5b63e76b2 ("arm: switch to RAW_COPY_USER")

Signed-off-by: Jonathan Liu <net147@gmail.com>
--- a/src/devicedrv/mali/linux/mali_osk_specific.h
+++ b/src/devicedrv/mali/linux/mali_osk_specific.h
@@ -17,7 +17,7 @@
 #ifndef __MALI_OSK_SPECIFIC_H__
 #define __MALI_OSK_SPECIFIC_H__
 
-#include <asm/uaccess.h>
+#include <linux/uaccess.h>
 #include <linux/platform_device.h>
 #include <linux/gfp.h>
 #include <linux/hardirq.h>
